core/tests/webdriverio_utils/forms.js

```diff
@@ -323,24 +323,32 @@ var UnicodeEditor = function(elem) {
 };
 
 var AutocompleteDropdownEditor = function(elem) {
-  var containerLocator = '.select2-container';
-  var searchInputLocator = '.select2-search input';
-  var dropdownElement = '.select2-dropdown';
+  var containerLocator = '.e2e-test-exploration-category-dropdown';
+  var searchInputLocator = '.mat-select-search-input.mat-input-element';
+  var categorySelectorChoice = '.e2e-test-exploration-category-selector-choice';
+
   return {
     setValue: async function(text) {
       await action.click('Container Element', elem.$(containerLocator));
```

Comment:
Why did you pass `true` here in the protractor code but not in webdriverio?

Reply:
i don't know why the click works in webdriverio code.